tests/pipeline/bismark_hisat/main.nf.test

```diff
@@ -0,0 +1,103 @@
+import com.askimed.nf.test.util.FileUtil;
+
+nextflow_pipeline {
+
+    name "Test Workflow main.nf"
+    script "main.nf"
+    profile "test,docker"
+    tag "pipeline"
```

Comment:
```suggestion
    tag "pipeline"
    tag "bismark_hisat"
```
Maybe?

Reply:
lets come back to this once we have a clear idea of how to use tags effectively